flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/strategy/AdaptiveBroadcastJoinOptimizationStrategy.java

```diff
@@ -162,7 +162,22 @@ protected void tryOptimizeAdaptiveJoin(
                             broadcastThreshold,
                             leftInputSize,
                             rightInputSize);
+                } else {
+                    LOG.info(
+                            "Modified stream edges for the join node [{}] failed.",
```

Comment:
maybe "Modification to stream edges for the join node [{}] failed. Keep the join node as is."

This is to prevent users from misinterpreting the situation as an unexpected error.